FinniversKit/Sources/Fullscreen/LoginEntryView/LoginEntryDialogueView.swift

```diff
@@ -115,7 +124,12 @@ class LoginEntryDialogueView: UIView {
             registerButton.topAnchor.constraint(equalTo: loginButton.bottomAnchor, constant: Warp.Spacing.spacing200),
             registerButton.leadingAnchor.constraint(equalTo: margins.leadingAnchor),
             registerButton.trailingAnchor.constraint(equalTo: margins.trailingAnchor),
-            registerButton.bottomAnchor.constraint(equalTo: margins.bottomAnchor),
+
+            vendLogoView.topAnchor.constraint(equalTo: registerButton.bottomAnchor, constant: Warp.Spacing.spacing400),
```

Comment:
Shouldn't it be `lessThanOrEqualToConstant `?
Have you tested on iPhone SE?

Reply:
on it!

Reply:
![Simulator Screenshot - smallscreen - 2025-05-07 at 12 52 41](https://github.com/user-attachments/assets/416b7172-4ecc-4d48-b95b-88c0b9405b61)

Reply:
What about Landscape? 👯  ahahah

Reply:
🔫 🔫 🔫 🔫 🔫

Reply:
![Uploading Screenshot 2025-05-07 at 13.21.44.png…]()

Reply:
<img width="911" alt="Screenshot 2025-05-07 at 13 21 44" src="https://github.com/user-attachments/assets/ec0fd22a-c78d-4af8-9c7c-f653fd9e1f0c" />

Reply:
Looks good to me 👍

Reply:
Omg, this is emo app... 🤣

Reply:
Don't listen to me